src/polytax/train.py

```diff
@@ -467,6 +466,7 @@ def __init__(self):
         super(Nanny, self).__init__()
         self.auto_setup()
         self.jobs = []
+        # Do we need original_sigint?
```

Comment:
I think so, but I haven't tested without this. I think we need to capture the original sigint otherwise it will continue to propagate.

Reply:
I was just a little confused as original_sigint seems to be unused